bedrock/products/templates/products/vpn/landing-refresh.html

```diff
@@ -407,4 +410,7 @@ <h3 class="c-learn-more-heading">
   {% endif %}
 
   {{ js_bundle('data_begincheckout') }}
+  {% if LANG == "en-US" and country_code == 'US' %}
```

Comment:
Since this is a short term experiment, we should probably put this behind a feature flag/switch so we can easily turn it on or off.

```suggestion
  {% if switch('experiment-ppa-na-vpn', ['en-US']) and country_code == 'US' %}
```
(or whatever switch name you prefer, just something unique and descriptive)